File: 백준/Bronze/2884. 알람 시계/알람 시계.py
import sys
input = sys.stdin.readline

H, M = map(int, input().split())

# 45분 전 알람
NH, NM = divmod(H * 60 + M - 45, 60)
if NH < 0:
    NH = 24 + NH

print(NH, NM)

File: 백준/Bronze/2884. 알람 시계/알람 시계.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split(' ');

let [h, m] = input.map(Number);

if (m - 45 < 0) {
  h -= 1;
  if (h < 0) {
    h += 24;
  }
  m = 60 + (m - 45);
} else {
  m -= 45;
}

console.log(h, m);
